common/beerocks/bcl/source/network/get_iface_stats.cpp

```diff
@@ -0,0 +1,243 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include <bcl/network/network_utils.h>
+
+#include <easylogging++.h>
+
+#include <errno.h>
+#include <linux/netlink.h>
+#include <linux/rtnetlink.h>
+#include <net/if.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#define IFLIST_REPLY_BUFFER 8192
+
+namespace beerocks {
+namespace net {
+
+/**
+ * @brief Netlink request type.
+ */
+struct nl_req_t {
+    /**
+     * Netlink message
+     */
+    struct nlmsghdr hdr;
```

Comment:
I see it throughout the file - you can drop the `struct` keyword since it's c++.  
though I can understand if you keep it to comply with the "standard" use of the no API's (which is C) 

```suggestion
    nlmsghdr hdr;
```

Reply:
Done